### PR TITLE
[FIX] website_sale[_loyalty]: fix disappearing code input 

### PR DESCRIPTION
**Pull request by anko-odoo**
If incorrect code was applied, possibility of putting another
code was blocked.

task-3601317

### CODE REVIEW

**Review by vava-odoo — APPROVED** [review]
Looks ok to me, thanks @anko-odoo 
Could you remove the comment, correct the typo in the commit title (and replace [] by () for the module)?

File: addons/website_sale_loyalty/views/website_sale_templates.xml [hunk 1]
```diff
@@ -146,8 +146,7 @@
     </template>
 
     <template id="reduction_coupon_code" inherit_id="website_sale.reduction_code">
-        <xpath expr="//t[@t-set='force_coupon']" position="replace">
-            <t t-set='force_coupon' t-value="website_sale_order.pricelist_id.code or request.params.get('code_not_available') or website_sale_order.get_promo_code_error(delete=False)"/>
```
Comment (Feyensv) [inline addons/website_sale_loyalty/views/website_sale_templates.xml:150]: Is it also necessary to drop the `get_promo_code_error` call ?
As it's the only place where it's called, I fear we are losing the support of an existing logic here :thinking:
If that's what we want, we should remove the logic fully in the fw-port PR in master.

Comment (anko-odoo) [reply addons/website_sale_loyalty/views/website_sale_templates.xml:150]: If I leave it, then it also not work properly. From what I see it's also called in line 20 and 22 to show the error message.

**Comment by anko-odoo** [comment]
@Feyensv Also it seems to me that in master that template can be deleted altogether as if I understand correctly _placeholder is for something that is no longer present so it has no use.

**Comment by Feyensv** [comment]
Little rebase for the ci/template please :pray: 
(also: dissapearing --> disappearing :eyes:)

**Review by Feyensv — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**